src/api/java/com/minecolonies/api/colony/IChunkmanagerCapability.java

```diff
@@ -61,7 +61,7 @@
         @Override
         public ChunkLoadStorage getChunkStorage(final int chunkX, final int chunkZ)
         {
-            return chunkStorages.get(new ChunkPos(chunkX, chunkZ));
+            return chunkStorages.remove(new ChunkPos(chunkX, chunkZ));
```

Comment:
Do we really want to remove it? Or give another acces to the list and have that remove it?